docs/sources/write/tooling-and-workflows/index.md

```diff
@@ -89,18 +92,18 @@ From github.com:grafana/mimir
  * [new branch]          threaded-reader                  -> origin/threaded-reader
 ```
 
-### Create a new branch from the main remote branch
+### Create a branch from the default remote branch
 
 By convention, the remote repository in GitHub is the source of truth for a repository's history.
-The main branch of a repository is typically called `main` and occasionally called `master`.
-Note that we prefer to use inclusive language, so `main` is the preferred name.
+The default branch of a repository is typically called `main` and occasionally called `master`.
+Note that Grafana prefers to use inclusive language, so `main` is the preferred name.
```

Comment:
Should we format this as a Note vs "Note that...."?  Or Just state "Grafana prefers to use...."

Reply:
I don't think it justifies a full note so:

```suggestion
Grafana prefers to use inclusive language, so `main` is the preferred name.
```